sdk/search/azure-search/src/main/java/com/azure/search/models/RegexFlags.java

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/**
+ * Defines values for RegexFlags.
+ */
+public final class RegexFlags extends ExpandableStringEnum<RegexFlags> {
```

Comment:
This class is missing a method to combine two `RegexFlags` together. This is accomplished via operator overloading in the .NET SDK:

https://github.com/Azure/azure-sdk-for-net/blob/8f8b4f5452c0521bfb71636730c947000e44b5f8/sdk/search/Microsoft.Azure.Search.Service/src/Customizations/Indexes/Models/RegexFlags.cs#L80

Without this, it's at best highly inconvenient to use regex analyzer functionality. We need to fix this before preview.

Reply:
Since this is a generated class, any change should be done with autorest config.